pfcli/main.py

```diff
@@ -94,3 +113,13 @@ def passwd(
     client.change_password(old_password, new_password)
 
     typer.secho("Password is changed successfully!", fg=typer.colors.BLUE)
+
+
+def _validate(_, token: str = typer.Option(..., prompt="Enter Code")):
+    r = requests.post(get_auth_uri("pf_user/self_signup/confirm"), json={"email_token": token})
+    try:
+        r.raise_for_status()
+        typer.echo("\n\nVerified!")
+        typer.echo("Sign up success! Please sign in.")
+    except HTTPError:
+        secho_error_and_exit("Invalid code... Please Try again.")
```

Comment:
`HTTPError` is not always related to invalid code, but it also related to duplicate email or username.
When duplicate email address is used, the error response code is 409 and the content is as follows:

```
'{"error":"not_unique_field","error_description":"Field should be unique"}'
```